stl/inc/xstring

```diff
@@ -3116,11 +3120,7 @@ private:
             _Right_data._Bx._Ptr = nullptr;
             _Swap_proxy_and_iterators(_Right);
         } else { // copy small string buffer
-#if _HAS_CXX20
-            if (_STD is_constant_evaluated()) { // begin the lifetime of the array elements before copying into them
-                _Construct_in_place(_Mypair._Myval2._Bx);
-            }
-#endif // _HAS_CXX20
+            _Mypair._Myval2._Activate_SSO_buffer();

```

Comment:
This should use `_My_data`. (Partially pre-existing, since the `_Construct_in_place` call had the same issue.)